clickhouse/tests/docker/docker-compose.yaml

```diff
@@ -29,155 +29,57 @@ services:
-      nofile:
-        soft: 262144
-        hard: 262144
     ports:
       - "8131:8123"
       - "9004:9000"
-    volumes:
-      - ./override_configs/zookeeper.xml:/etc/clickhouse-server/config.d/zookeeper.xml
-      - ./override_configs/remote_servers.xml:/etc/clickhouse-server/config.d/remote_servers.xml
-      - ./override_configs/compression.xml:/etc/clickhouse-server/config.d/compression.xml
-      - ./metrika.xml:/etc/metrika.xml
-      - ./macros/macros-04.xml:/etc/clickhouse-server/config.d/macros.xml
-      - ./users.xml:/etc/clickhouse-server/users.d/users.xml
-      - ./init.sql:/docker-entrypoint-initdb.d/init.sql
-    depends_on:
-      - clickhouse-zookeeper
-    networks:
-      - clickhouse-network
 
   clickhouse-05:
-    image: ${CLICKHOUSE_REPOSITORY}:${CLICKHOUSE_VERSION}
+    << : *ch-node
     hostname: clickhouse-05
     container_name: clickhouse-05
-    cap_add:
-      - IPC_LOCK
-      - NET_ADMIN
-    ulimits:
-      nproc: 65536
-      nofile:
-        soft: 262144
-        hard: 262144
     ports:
       - "8132:8123"
       - "9005:9000"
-    volumes:
-      - ./override_configs/zookeeper.xml:/etc/clickhouse-server/config.d/zookeeper.xml
-      - ./override_configs/remote_servers.xml:/etc/clickhouse-server/config.d/remote_servers.xml
-      - ./override_configs/compression.xml:/etc/clickhouse-server/config.d/compression.xml
-      - ./metrika.xml:/etc/metrika.xml
-      - ./macros/macros-05.xml:/etc/clickhouse-server/config.d/macros.xml
-      - ./users.xml:/etc/clickhouse-server/users.d/users.xml
-      - ./init.sql:/docker-entrypoint-initdb.d/init.sql
-    depends_on:
-      - clickhouse-zookeeper
-    networks:
-      - clickhouse-network
 
   clickhouse-06:
-    image: ${CLICKHOUSE_REPOSITORY}:${CLICKHOUSE_VERSION}
+    << : *ch-node
     hostname: clickhouse-06
     container_name: clickhouse-06
-    cap_add:
-      - IPC_LOCK
-      - NET_ADMIN
-    ulimits:
-      nproc: 65536
-      nofile:
-        soft: 262144
-        hard: 262144
     ports:
       - "8133:8123"
       - "9006:9000"
-    volumes:
-      - ./override_configs/zookeeper.xml:/etc/clickhouse-server/config.d/zookeeper.xml
-      - ./override_configs/remote_servers.xml:/etc/clickhouse-server/config.d/remote_servers.xml
-      - ./override_configs/compression.xml:/etc/clickhouse-server/config.d/compression.xml
-      - ./metrika.xml:/etc/metrika.xml
-      - ./macros/macros-06.xml:/etc/clickhouse-server/config.d/macros.xml
-      - ./users.xml:/etc/clickhouse-server/users.d/users.xml
-      - ./init.sql:/docker-entrypoint-initdb.d/init.sql
-    depends_on:
-      - clickhouse-zookeeper
-    networks:
-      - clickhouse-network
 
   clickhouse-zookeeper:
     # Using zookeeper `3.6.0` is raising this error:
     #   Connection request from old client /172.30.0.7:51646; will be dropped if server is in r-o mode
     # For now, we use zookeeper `3.5.7` instead
     # Similar case: https://stackoverflow.com/a/22884698
+    platform: linux/x86_64
```

Comment:
QQ: Why is this needed?

Reply:
ah, it crept in while I was running locally, I'll remove it.